### DISCUSSION TITLE
default_2fa_method per user

### DISCUSSION THREAD

**Discussion by Nicolas-mahe** [authelia/authelia#7612] [Q&A] [answered]
Hello, I have Authelia running with Swag reverse proxy, both on docker and latest version.
I ask me, and I don't find this in documentation, Can I set a default A2F method ( I use webauthn and Totp ) per user ?
Example, I have a Yubi key so I want authelia prompt me webauthn method, but for an other user who don't have a Yubikey, Authelia prompt him TOTP methode. ( I use local user management ) 
Thanks in advance

**Comment by james-d-elliott** [comment, marked as answer]
When a user selects a method it becomes their default for that browser, they can adjust it in their own settings if they want for all browsers which do not already have a preference.

**Reply by Nicolas-mahe** [reply]
Oh yes, I forgot there is a menu! thanks :)
